### PR TITLE
Feat/archive note

### PR DESCRIPTION
**Pull request by viru2001**
- user can archive note from the notes page
- user can unarchive note from the archive page
- user can permanently delete note from archive page

archive note

### CODE REVIEW

**Review by tushargoyal812 — COMMENTED** [review]
code is clean and to the point.
naming convention is very.
liked how you structured files,good work.

File: src/frontend/utils/archiveNotesUtils/addToArchives.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { addToArchiveService } from "frontend/services";
+
+const addToArchives = async (token, note, notesDispatch) => {
+  await addToArchiveService(token, note);
+  notesDispatch({ type: "ADD_TO_ARCHIVE", payload: note });
+};
+export { addToArchives };
```
Comment (SHIKHASINGH1506) [inline src/frontend/utils/archiveNotesUtils/addToArchives.js:7]: You could have created one file for archive service and add all the crud apis to it, as "addToArchiveService" doesn't sound like a method name. Just a suggestion.

**Review by SHIKHASINGH1506 — APPROVED** [review]
Left one suggestion, 
folder structure looks nice.

File: src/frontend/utils/archiveNotesUtils/addToArchives.js (hunk 1 above)
Comment (viru2001) [reply src/frontend/utils/archiveNotesUtils/addToArchives.js:7]: okay. will think on it

**Review by vishalg8454 — COMMENTED** [review]
Learnt the new way of calling API's via service functions. Nice work.

**Pull request merged**